drivers/watchdog/Kconfig.ifx_cat1

```diff
@@ -0,0 +1,14 @@
+# Infineon CAT1 Watchdog configuration options
+
+# Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config WATCHDOG_INFINEON_CAT1
+	bool "Infineon CAT1 Watchdog Driver"
```

Comment:
Almost others starts with "WDT_" so I think it's better to change WDT_INFINEON_CAT1.

Reply:
updated..